meraki_appliance.tf

```
@@ -4,7 +4,7 @@ locals {
       for organization in try(domain.organizations, []) : [
         for network in try(organization.networks, []) : {
           key                    = format("%s/%s/%s", domain.name, organization.name, network.name)
-          network_id             = meraki_network.organizations_networks[format("%s/%s/%s", domain.name, organization.name, network.name)].id
+          network_id             = local.network_ids[format("%s/%s/%s", domain.name, organization.name, network.name)].id
```

`network_ids[]` has the ID, not the whole network, so this should be
```suggestion
          network_id             = local.network_ids[format("%s/%s/%s", domain.name, organization.name, network.name)]
```